### ISSUE TITLE
The library validates everything correctly

### ISSUE THREAD

**Issue by PatriceVignola** [hapijs/isemail#174]
I am using node v8.10.0. This is my first time using this library so I don't know if it's working on older node versions, but when I'm running the examples, everything validates correctly: 

```js
$ node
> var Isemail = require('isemail');
undefined
> Isemail.validate('test@iana.org');
true
> Isemail.validate('test@iana.org', {errorLevel: true});
0
> Isemail.validate('test@e.com', {errorLevel: true});
0
> Isemail.validate('test@e.com', {errorLevel: 7});
0
> Isemail.validate('test@e.com', {errorLevel: 6});
0
```

I also did more tests, and basically everything that has a "@" validates, even without domain names.

**Comment by WesTyler** [comment]
All of those emails are valid email addresses, per the RFCs that govern email rules.

"Valid email" !== "deliverable email"

If you can provide an example that passes, along with an RFC reference that indicates it shouldn't, I will reopen.

**Comment by PatriceVignola** [comment]
@WesTyler Like I said, it's my first time using this library, but the code I pasted here is the same code found int the "Examples" section of the README, except it doesn't give me the same answers:

```js
$ node
> var Isemail = require('isemail');
undefined
> Isemail.validate('test@iana.org');
true
> Isemail.validate('test@iana.org', {errorLevel: true});
0
> Isemail.validate('test@e.com', {errorLevel: true});
6
> Isemail.validate('test@e.com', {errorLevel: 7});
0
> Isemail.validate('test@e.com', {errorLevel: 6});
6
```

I was just pointing out that there is a bug somewhere, either in:
- The documentation (maybe outdated? It could be fixed pretty quickly)
- The code
- The dependencies

I think it's worth reopening the issue, even if it's just to fix the documentation.

**Comment by WesTyler** [comment]
Okay, I'm with you now. The original issue just said "everything validates correctly", but no reference to the documentation. 

I've confirmed that the `@e.com` validations provided in the documentation do return `0` instead of `6` as indicated in the docs. I believe this is a documentation issue

**Comment by skeggse** [comment]
Chiming in here: we should probably add a good example of a bad email address. The example in the readme never worked, and I probably goofed and copied the errorLevel for the old asynchronous version of the function that performed a dns check.

**Comment by WesTyler** [comment]
I just opened up a PR that includes a "bad email" address.

I used one with a low error level (in the "valid for SMTP but has unusual elements" range) to show the effects of the errorLevel flags without people being potentially confused by something like `example@` that has an error level of 131.

The Travis build for that PR is failing on the `node` engine, because of lab not yet being compatible with Node 10.
